### ISSUE TITLE
Revise DOI and ARXIV sanitation URLS

### ISSUE THREAD

**Issue by marshalmiller** [rottingresearch/rottingresearch#59]
Right now they are pulling URLS with doi.org or arxiv.org. The concern would be that it would have a false negative if used in Internet Archive URLs that contain the URL that is archived. I will Address ASAP.

**Comment by marshalmiller** [comment]
@Aakash-pal Yes, this would be a move in the right direction. There is also an issue where the style of citation changes and so we need to add more possible patterns. One example would be DOIs. You have the current best practice, but there was a time when the proper citation was considered simply, "DOI: 10.1007/s11089-018-0833-1". It might make sense to also indicate in the report that this is an outdated way to cite DOIs? Would be a different feature, though. Maybe all of these shoudl be seperate issues. Either way, you have solved the problem as described.

**Comment by Aakash-pal** [comment]
Hi @marshalmiller ,
Have you had the chance to review the issue.
